modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -87,10 +88,15 @@ function ManagementToolbar({
 		() => normalizeDropdownItems(viewTypeItems),
 		[viewTypeItems]
 	);
-	const viewTypeIcon = useMemo(
-		() => viewTypeItems?.find((item) => item.active)?.icon,
+	const activeViewType = useMemo(
+		() => viewTypeItems?.find((item) => item.active),
 		[viewTypeItems]
 	);
+	const viewTypeIcon = activeViewType.icon;
```

Comment:
I'd not extract this to a variable, I'd just use `activeViewType?.icon` where you need to.

Reply:
Since the CI is already running, I prefer to send a follow up PR once merged if necessary